src/Select.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import styled from 'styled-components'
 import TextInputWrapper, {StyledWrapperProps} from './internal/components/TextInputWrapper'
+import FormControl from './FormControl/FormControl'
```

Comment:
I think we definitely prefer `FormControl.autowirable` and that's the only way it's accessible via the public API. There are just two internal components (radio & checkbox) that are still referenced inside the body of `FormControl` so I had to avoid the circular reference.

I might dig into why those two still need to be explicitly referenced though -- maybe that can be abstracted as well.